packer/vagrant-box.json

```diff
@@ -151,7 +236,25 @@
                 "scripts/install-certstrap.sh",
                 "scripts/usr-local-bin-to-path.sh",
                 "scripts/create-kube-certs.sh",
-                "scripts/create-kubedns.sh",
+                "scripts/create-kubedns.sh"
+            ],
+            "start_retry_timeout": "7m"
+        },
+        {
+            "type": "shell",
+            "only": ["triton"],
+            "inline": [
+                "# Install helm",
+                "wget -O - https://kubernetes-helm.storage.googleapis.com/helm-v2.5.1-linux-amd64.tar.gz | tar xz -C /usr/local/bin --no-same-owner --strip-components=1 linux-amd64/helm",
+                "sudo -i -u vagrant helm init",
+                "# Install additional packages",
+                "zypper --non-interactive install jre-openjdk-headless make zip"
```

Comment:
What is the jre-openjdk-headless dep needed for? Also, `make` would be needed for dev deployments, but would CI deployments need it?

Reply:
It's needed for Jenkins to run the slave bits. (After all, Jenkins is a giant Java app.)

Yes, CI needs `make` because it builds the images (`make vagrant-prep`, in fact).

Reply:
hmm... have you tested this? I wouldn't expect this to work without dev tools. I thought the idea was that for CI, the image we would use wouldn't need to `make vagrant-prep`, just take the pre-generated helm charts and deploy them.

Reply:
Yes; I've got successful builds (heck, even 🚬 ) from it.  The Jenkins slaves here are doing double duty, both to generate the helm charts / images, as well as grabbing them (possibly on a different slave) to test them.  We don't really need dev tools, because we build everything in docker images (which themselves have the dev tools installed).

Reply:
if the `vagrant-prep` is happening inside containers.. why do we need make?

Reply:
No, the `vagrant-prep` happens directly on the VM without any containers. The actual package building happens inside containers (spawned via `make` → `fissile` → `docker`), hence not needing dev tools.

Reply:
how are `make vagrant-prep` and `make run` working without direnv?

Reply:
[Sourcing .envrc directly](https://github.com/SUSE/scf/blob/9059d23cc0e59bcf5474181fdbe974acfd6d0ea9/Jenkinsfile#L140)